# None

## Input

_Title: Fungal, bacterial, and archaeal communities mediating C cycling_

_and trace gas flux in peatland ecosystems subject to climate change_

__

_Abstract:_

_Metagenome and metatranscriptome analyses. In the second part of this study we will conduct in-depth analysis to integrate metagenomics and metatranscriptomics studies that will leverage two i_

_ntensive peatland climate change experiments: PEATcosm (USDA & NSF funded) and SPRUCE (DOE funded). The PEATcosm experiment is investigating the effects of water table and peatland vegetation on trace gas flux in bog ecosystems. Dominant vascular plants (heath and sedge family) are hypothesized to be key ecosystem engineers that influence microbial community structure both via symbiotic associations that have strong extracellular enzyme activity (Ericaeae), and by transport of oxygen and labile C into the peat (Carex).  We will investigate 8 microenvironments: Ericaceae and sedge rhizosphere peat, at two depths, in both high and low water manipulative treatments simulating anticipated climate-mediated changes in water balance. We anticipate collecting 3 replicate samples per sample type to be used for both metagenome and metatranscriptome seque_

_ncing; each set of 3 could be pooled on a single lane of Illumina HiSeq, bringing the total to 16 lanes. The SPRUCE experiment is designed to understand threshold level responses of a peatland system to experimental warming.  It features fourteen plots that will be heated over a temperature gradient of 0 to +9 Â°C above ambient, factorial with CO2 enrichment over a 10yr period.  The experiment is being constructed in Minnesota with heating scheduled to begin in spring of 2014. Pretreatment environmental, vegetation and biogeochemical data has already been collected over the last 3 years.  To complement such efforts, we are requesting analysis from 70 samples representing spatial and depth variation across the bog (14 plots/5 depths). Our goal is to provide baseline data on community functional genomic potential (metagenomic DNA) and their activities (metatranscriptomic RNA) in order to guide future intensive investigations at this site.  We anticipate multiplexing of up to 10 DNA or RNA samples per lane of the HiSeq for a total of 14 lanes of capacity for this part_

__

## Results



### environments


- None Ericaceae and sedge rhizosphere peat

- bog ecosystems bog ecosystems

- Heath and Sedge Family Ecosystems heath and sedge family ecosystems

- SPRUCE experiment SPRUCE experiment

- 14 plots 14 plots

- depths 5 depths


### causal_relationships


- cause:
  - Water Table water table

- effect:
  - Trace Gas Flux trace gas flux


- cause:
  - Peatland vegetation Peatland vegetation

- effect:
  - Microbial community structure Microbial community structure


- cause:
  - None oxygen and labile C

- effect:
  - peat ENVO:00005774



### variables


- Water Table water table

- Peatland vegetation peatland vegetation

- oxygen CHEBI:15379

- labile C labile C


### treatments


- water manipulative treatments water manipulative treatments

- CO2 enrichment CO2 enrichment


### measurements


- value:
  - pretreatment

- unit:
  - Environmental Data environmental data


- value:
  - biogeochemical

- unit:
  - data http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C25474


- value:
  - functional genomic potential

- unit:
  - community http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C16453


- value:
  - activities

- unit:
  - None



### location


- Minnesota Minnesota


### environmental_material


- Trace Gas Flux trace gas flux


### sequencing_technologies


- Illumina HiSeq Illumina HiSeq


### organisms


- fungi NCBITaxon:4751

- bacteria NCBITaxon:2

- archaea NCBITaxon:2157

- Ericaeae Ericaeae

- Carex NCBITaxon:13398



YAML:

<details>
```yaml
input_id: null
input_text: "Title: Fungal, bacterial, and archaeal communities mediating C cycling\n\
  and trace gas flux in peatland ecosystems subject to climate change\n\nAbstract:\n\
  Metagenome and metatranscriptome analyses. In the second part of this study we will\
  \ conduct in-depth analysis to integrate metagenomics and metatranscriptomics studies\
  \ that will leverage two i\nntensive peatland climate change experiments: PEATcosm\
  \ (USDA & NSF funded) and SPRUCE (DOE funded). The PEATcosm experiment is investigating\
  \ the effects of water table and peatland vegetation on trace gas flux in bog ecosystems.\
  \ Dominant vascular plants (heath and sedge family) are hypothesized to be key ecosystem\
  \ engineers that influence microbial community structure both via symbiotic associations\
  \ that have strong extracellular enzyme activity (Ericaeae), and by transport of\
  \ oxygen and labile C into the peat (Carex).  We will investigate 8 microenvironments:\
  \ Ericaceae and sedge rhizosphere peat, at two depths, in both high and low water\
  \ manipulative treatments simulating anticipated climate-mediated changes in water\
  \ balance. We anticipate collecting 3 replicate samples per sample type to be used\
  \ for both metagenome and metatranscriptome seque\nncing; each set of 3 could be\
  \ pooled on a single lane of Illumina HiSeq, bringing the total to 16 lanes. The\
  \ SPRUCE experiment is designed to understand threshold level responses of a peatland\
  \ system to experimental warming.  It features fourteen plots that will be heated\
  \ over a temperature gradient of 0 to +9 \xB0C above ambient, factorial with CO2\
  \ enrichment over a 10yr period.  The experiment is being constructed in Minnesota\
  \ with heating scheduled to begin in spring of 2014. Pretreatment environmental,\
  \ vegetation and biogeochemical data has already been collected over the last 3\
  \ years.  To complement such efforts, we are requesting analysis from 70 samples\
  \ representing spatial and depth variation across the bog (14 plots/5 depths). Our\
  \ goal is to provide baseline data on community functional genomic potential (metagenomic\
  \ DNA) and their activities (metatranscriptomic RNA) in order to guide future intensive\
  \ investigations at this site.  We anticipate multiplexing of up to 10 DNA or RNA\
  \ samples per lane of the HiSeq for a total of 14 lanes of capacity for this part\n"
input_title: null
named_entities:
- id: Ericaceae and sedge rhizosphere peat
  label: null
- id: bog ecosystems
  label: bog ecosystems
- id: heath and sedge family ecosystems
  label: Heath and Sedge Family Ecosystems
- id: SPRUCE experiment
  label: SPRUCE experiment
- id: 14 plots
  label: 14 plots
- id: 5 depths
  label: depths
- id: water table
  label: Water Table
- id: trace gas flux
  label: Trace Gas Flux
- id: Peatland vegetation
  label: Peatland vegetation
- id: Microbial community structure
  label: Microbial community structure
- id: oxygen and labile C
  label: null
- id: ENVO:00005774
  label: peat
- id: water table
  label: Water Table
- id: peatland vegetation
  label: Peatland vegetation
- id: CHEBI:15379
  label: oxygen
- id: labile C
  label: labile C
- id: water manipulative treatments
  label: water manipulative treatments
- id: CO2 enrichment
  label: CO2 enrichment
- id: environmental data
  label: Environmental Data
- id: http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C25474
  label: data
- id: http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C16453
  label: community
- id: Minnesota
  label: Minnesota
- id: trace gas flux
  label: Trace Gas Flux
- id: Illumina HiSeq
  label: Illumina HiSeq
- id: NCBITaxon:4751
  label: fungi
- id: NCBITaxon:2
  label: bacteria
- id: NCBITaxon:2157
  label: archaea
- id: Ericaeae
  label: Ericaeae
- id: NCBITaxon:13398
  label: Carex
raw_completion_output: '

  environments: Ericaceae and sedge rhizosphere peat; bog ecosystems; heath and sedge
  family ecosystems; SPRUCE experiment; 14 plots; 5 depths

  causal_relationships: effect of water table on trace gas flux; effect of peatland
  vegetation on microbial community structure; effect of oxygen and labile C on peat

  variables: water table; peatland vegetation; oxygen; labile C

  treatments: water manipulative treatments; CO2 enrichment

  measurements: pretreatment environmental data; biogeochemical data; community functional
  genomic potential; activities

  location: Minnesota

  environmental_material: trace gas flux

  sequencing_technologies: Illumina HiSeq

  organisms: fungi; bacteria; archaea; Ericaeae; Carex'
results:
  causal_relationships:
  - cause: water table
    effect: trace gas flux
  - cause: Peatland vegetation
    effect: Microbial community structure
  - cause: oxygen and labile C
    effect: ENVO:00005774
  environmental_material:
  - trace gas flux
  environments:
  - Ericaceae and sedge rhizosphere peat
  - bog ecosystems
  - heath and sedge family ecosystems
  - SPRUCE experiment
  - 14 plots
  - 5 depths
  location:
  - Minnesota
  measurements:
  - unit: environmental data
    value: pretreatment
  - unit: http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C25474
    value: biogeochemical
  - unit: http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C16453
    value: functional genomic potential
  - unit: null
    value: activities
  organisms:
  - NCBITaxon:4751
  - NCBITaxon:2
  - NCBITaxon:2157
  - Ericaeae
  - NCBITaxon:13398
  sequencing_technologies: Illumina HiSeq
  treatments:
  - water manipulative treatments
  - CO2 enrichment
  variables:
  - water table
  - peatland vegetation
  - CHEBI:15379
  - labile C

```

</details>


Completion:

<details>
```

environments: Ericaceae and sedge rhizosphere peat; bog ecosystems; heath and sedge family ecosystems; SPRUCE experiment; 14 plots; 5 depths
causal_relationships: effect of water table on trace gas flux; effect of peatland vegetation on microbial community structure; effect of oxygen and labile C on peat
variables: water table; peatland vegetation; oxygen; labile C
treatments: water manipulative treatments; CO2 enrichment
measurements: pretreatment environmental data; biogeochemical data; community functional genomic potential; activities
location: Minnesota
environmental_material: trace gas flux
sequencing_technologies: Illumina HiSeq
organisms: fungi; bacteria; archaea; Ericaeae; Carex
```

</details>
